app/services/solr_service.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class SolrService
+  def self.connection
+    solr_core = ENV["SOLR_CORE"] ||= "blacklight-core"
+    solr_host = ENV["SOLR_HOST"] ||= "http://localhost:8983/solr"
+    RSolr.connect url: "#{solr_host}/#{solr_core}"
```

Comment:
this will make ugly stuff if the "solr_host" value has an ending slash, consider using 

URI.join('http://foo/', '/bar/', '/baz').to_s

Reply:
Unfortunately this ends up leaving out '/bar/ in all the ways I've tried it.

Reply:
lol, I guess I should read the docs first